core/profile.c

```diff
@@ -1660,7 +1660,7 @@ void compare_samples(struct plot_data *e1, struct plot_data *e2, char *buf, int
 			int sac = lrint(volume_used / atm * 60 / delta_time);
 			memcpy(buf2, buf, bufsize);
 			volume_value = get_volume_units(sac, &volume_precision, &volume_unit);
-			snprintf_loc(buf, bufsize, translate("gettextFromC", "%s SAC: %.*f%s"), buf2, volume_precision, volume_value, volume_unit);
+			snprintf_loc(buf, bufsize, translate("gettextFromC", "%s SAC:%.*f%s/min"), buf2, volume_precision, volume_value, volume_unit);
 		}
```

Comment:
could you please explain your motivation for removing the space after the `:` in these two instances.
the rest of the file seems to use space after the `:` character more consistently.

Reply:
@neolit123 
Yes, you are right: In the profile info box we have "name: xx.xvalue" with space consistently.
In this "delta sample string" we had a mix up to now with space (2 occurrences) and w/o space (8 occurrences). I guess the reason why w/o space may be preferred here is to limit the length of the string as much as possible.
I think that this small mismatch between layout in profile info box and this delta sample string is ok. I'll show you one picture:
![grafik](https://user-images.githubusercontent.com/25316050/37017735-03b06786-2112-11e8-9187-232dd8028233.png)

Reply:
ok, understood.